test/Release/MysqlReleaseToggleCollectionGatewayTest.php

```diff
@@ -287,4 +132,120 @@ private function addToggleMarketingInformationToDatabase( $toggleId, $screenshot
             'app_notification_copy_text' => $appNotificationCopyText
         ] );
     }
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $bootstrap = new Bootstrap();
```

Comment:
Wait :D I think I missed this at the last merge, but I thought we aggreed that the Bootsrap will be only initialized once in bootstrap.php. Here you must use Bootstrap::getInstance(), or else bootstrap.php worth nothing :D